.github/workflows/template-connect-popup-test-params.yml

```diff
@@ -92,16 +100,24 @@ jobs:
         run: echo "branch=${GITHUB_HEAD_REF:-${GITHUB_REF#refs/heads/}}" >> $GITHUB_OUTPUT
         id: extract_branch
 
-      - name: Build webextension
+      - name: Install dependencies
         run: |
           yarn install --immutable
           yarn build:libs
```

Comment:
we should use directly building just the necessary lib, not all of them, right?

Reply:
yes, that shouldn't be needed globally here.

Reply:
You are right, that `yarn build:libs` was totally unnecessary, done  https://github.com/trezor/trezor-suite/pull/12011/commits/1c66aef5021ddcd26a75bc52924455b99025cb39